xLights/TabSequence.cpp

```diff
@@ -317,6 +315,22 @@ wxString xLightsFrame::LoadEffectsFileNoCheck()
         UnsavedRgbEffectsChanges = true;
     }
     FseqDir = fseqDirectory;
+    xsqDirectory = GetXmlSetting("xsqDir", showDirectory);
+    ObtainAccessToURL(xsqDirectory);
+    if (!wxDir::Exists(xsqDirectory)) {
```

Comment:
So if I read this right if the setting the user entered for where they want their XSQ files stored cannot be found then its going to just change the registry?  Seems like some folks are going to use cloud drives which could potentially have connection issues and they wouldn't want their setting changed due to connection issues.

Also the second if statement is always true if the code goes into the first if statement so it could probably be an "else if".